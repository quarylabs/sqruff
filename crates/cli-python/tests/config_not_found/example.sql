select foo.bar from table1 foo;

