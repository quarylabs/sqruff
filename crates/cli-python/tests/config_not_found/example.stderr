The specified config file 'tests/config_not_found/non_existant.cfg' does not exist.
